README.md

```diff
@@ -43,23 +43,31 @@ import com.anthonycr.mockingbird.core.Verify
 import com.anthonycr.mockingbird.core.fake
 import com.anthonycr.mockingbird.core.verify
 
-interface MyInterface {
-  fun doThing(value: String)
+interface Analytics {
+    fun trackEvent(event: String)
 }
 
-class MyTest {
-  @Verify
-  private val myInterface: MyInterface = fake()
+class ClassToTest(private val analytics: Analytics) {
+    fun doSomething() {
+        analytics.trackEvent("doSomething was called!")
+    }
+}
```

Comment:
I've been debating about adding more inline comments here to explain this further but right now I think this is fine as is?

Reply:
I think it's fine as is. I think it's safe to assume for now that anyone reading this readme has already looked at the likes of mockito and mockk and knows the mock/verify pattern.